### PR TITLE
Refactor from single page app to multipage using React Router

### PR DESCRIPTION
**Pull request by timothytamnguyen**
- app and navbar are the main bulk of changes

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -102,34 +107,55 @@ class App extends React.Component {
...
+          />
+          <Route
+            path="/profile"
+            render={() => (
+              <ProfilePage user={user} />
+            )}
+          />
+        </Switch>
+      </Router>
     );
```
Comment (chiefmikey) [inline client/src/components/App.jsx:159]: look at all these chickens

**Review by tillmattson — APPROVED** [review]
Looks good.

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createBrowserHistory } from 'history';
 import App from './components/App';
 import './assets/styles.scss';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const originalHistory = createBrowserHistory();
+
+ReactDOM.render(<App history={originalHistory} />, document.getElementById('root'));
```
Comment (chiefmikey) [inline client/src/index.jsx:9]: this is pretty interesting

**Review by chiefmikey — APPROVED** [review]
swag level = <em>100</em>

**Pull request merged**